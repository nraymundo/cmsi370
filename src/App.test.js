import React from "react";
import ReactDOM from "react-dom";
import TestRenderer from "react-test-renderer";
import App from "./App";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("should render App", () => {
  const component = TestRenderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
