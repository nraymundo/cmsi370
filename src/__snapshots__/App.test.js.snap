// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App 1`] = `
<div
  className="App"
>
  <div
    className="AppHeader"
  >
    <h1>
      Take That For Data
    </h1>
    <h2>
      Premier League | Bundesliga | Serie A | La Liga | Ligue 1
    </h2>
  </div>
  <form
    className="SearchForm"
    onSubmit={[Function]}
  >
    <div
      className="SearchBar"
    >
      <input
        autoComplete="off"
        id="search"
        name="query"
        onChange={[Function]}
        placeholder="Team or league"
        type="text"
        value=""
      />
      <button
        disabled={true}
        id="searchButton"
        type="submit"
      >
        Search
      </button>
    </div>
    <div
      className="SearchResults"
    />
  </form>
</div>
`;
